index.html

```diff
@@ -446,6 +450,12 @@ <h1>Terminology</h1>
       <dd>Mechanisms defined by WoT for distributing and accessing
         <a>WoT Thing Descriptions</a> on the network, 
         either locally or remotely.</dd>
+      <dt>
+        <dfn data-lt="WoT Discoverer">Discoverer</dfn>
+      </dt>
+      <dd>An entity that generates and registers a TD with an exploration service
+        on behalf of a <a>Thing</a> which may not be able to do it for itself.
+      </dd>
```

Comment:
I still believe that the term Discoverer is misleading; see https://github.com/w3c/wot-discovery/issues/216#issuecomment-923942848
Maybe it is worth reconsidering it as part of the migration.